code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -34,8 +34,11 @@
 			src << "<span class='warning'>Your powers are useless on this holy ground.</span>"
 			return 0
 	if(check_holy(src) && !fullpower)
-		var/turf/simulated/T = get_turf(src)
-		if(T.lighting_lumcount > 2)
+		var/atom/movable/lighting_overlay/L
+		for(var/atom/movable/lighting_overlay/LF in get_turf(src))
```

Comment:
According to aran when he was trying to make get_turf or get_area quicker he found that locate was slower than the recursive loop